.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "chrome",
+      "request": "launch",
+      "name": "Debug happa",
+      "url": "http://localhost:7000",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```

Comment:
JSON does not allow comments. This is technically invalid, hence all the red highlighting here in GitHub.